src/components/SummaryKeyComponents/SemScholarAbstractAndTldrComponent.js

```diff
@@ -0,0 +1,55 @@
+import { getPaperTldrAndAbstract } from "../../services/SemanticScholarService";
+
+/**
+ * Class representing the Abstract and the TLDR Accordion items
+ * @property {Object} components object that holds DOM elements that represent this component, as well as component's context
+ * @property {HTMLElement} components.abstractText html p element that represents the text of the accordion item
+ * @property {HTMLElement} components.tldrText html p element that represents the text of the accordion item
+ */
+class SemScholarAbstractAndTldrComponent {
```

Comment:
Tbh i dont think that this should be a component because to me logically it does not make sense to contain together 2 out of 4 components that on the first look look basically the same together. I could be wrong but maybe it would be better to keep tldrText and abstractText in SummaryKeyComponent and rename loadData method and keep it in SummarizerService.